src/service/auth/sendMail.ts

```
@@ -0,0 +1,48 @@
+import { Request, Response } from 'express';
+import { createTransport } from 'nodemailer';
+import { SendMailRequest } from '../../types/auth';
+import { BasicResponse } from '../../types';
+
+const emailId = process.env.EMAIL_ID;
+const emailPw = process.env.EMAIL_PW;
+if (!emailId || !emailPw) {
+  throw Error('env 변수 불러오기 실패패');
+}
+
+export const sendMail = async (req: Request<{}, {}, SendMailRequest>, res: Response<BasicResponse>) => {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)